src/Crypto/JWT.hs

```diff
@@ -323,6 +333,18 @@ validateExpClaim conf (ClaimsSet _ _ _ (Just e) _ _ _ _) = do
     else throwError (review _JWTExpired ())
 validateExpClaim _ _ = pure ()
 
+validateIatClaim
+  :: (MonadTime m, HasCheckIssuedAt a, HasAllowedSkew a, AsJWTError e, MonadError e m)
+  => a
+  -> ClaimsSet
+  -> m ()
+validateIatClaim conf (ClaimsSet _ _ _ _ _ (Just e) _ _) = do
+  now <- currentTime
+  if ((view checkIssuedAt conf) && (view _NumericDate e) > addUTCTime (abs (view allowedSkew conf)) now)
```

Comment:
It might be nicer to use [`Control.Monad.when`](http://hackage.haskell.org/package/base-4.9.0.0/docs/Control-Monad.html#v:when) here, e.g.:

```haskell
when (view checkIssuedAt conf) $
  when (view _NumericDate e > addUTCTime (abs (view allowedSkew conf)) now) $
    throwError (review _JWTIssuedAtFuture ())
```